src/Symfony/Component/Process/Process.php

```diff
@@ -307,7 +307,6 @@ public function start(callable $callback = null)
         }
 
         $this->updateStatus(false);
-        $this->checkTimeout();
```

Comment:
This change is not actually needed for fixing the bug, but I thought this line is useless.